package.json

```diff
@@ -52,12 +61,12 @@
     "menus": {
       "editor/context": [
         {
-            "when": "editorHasSelection && resourceLangId == justAntlr",
+            "when": "editorHasSelection",
```

Comment:
I don't think it's necessary to restrict the language because I might work with other Antlr plugins, so I don't have to switch languages repeatedly

Just check the file suffix in the command